### ISSUE TITLE
🦺[Tech Debt] Add ability for developers to compile without private data

### ISSUE THREAD

**Issue by kcw-grunt** [litecoin-foundation/litewallet-ios#48]
## Why?
Others trying to compile the project will not be able to test without making this fix.

**Comment by JCHeist** [comment]
https://github.com/litecoin-foundation/litewallet-ios/tree/techdebt/overriding-private-keys-files

Here is the solution I implemented instead of just editing the main Target's Project file to not include the private key files. We may not want to source control this file but im not sure the most elegant way to handle this. Any ideas? @kcw-grunt

**Comment by kumarlav0** [comment]
I followed your Steps which you mentioned in  url: 
https://github.com/litecoin-foundation/litewallet-ios/tree/techdebt/overriding-private-keys-files
 
Step by Step I did. But when I launch the project. I got an error every time. 
ERROR:  

The remote repository could not be accessed. Make sure a valid repository exists at the specified location and that the correct credentials have been supplied.



![Screenshot 2021-05-25 at 4 53 09 PM](https://user-images.githubusercontent.com/58595312/119489794-b2990f80-bd79-11eb-965d-bd48f5f0b58a.png)
